cli/src/commands/tag.ts

```diff
@@ -0,0 +1,44 @@
+import { Command, flags } from "@oclif/command";
+import { hashContractDefinition } from "../../../lib/src/tagging/hash";
+import { safeParse } from "../common/safe-parse";
+
+const ARG_API = "spot_contract";
+
+/**
+ * oclif command to generate a tag based on a Spot contract
+ */
+export default class Tag extends Command {
```

Comment:
Tag doesn't sound like it describes this feature. How about something similar to `Checksum`

Reply:
No worries Leslie. I can find a more appropriate name. Will address.

Reply:
No worries Leslie. I can find a more appropriate name. Will address.